code/api/src/modules/user/model.js

```diff
@@ -14,9 +14,6 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
-    },
-    description: {
-      type: DataTypes.TEXT
```

Comment:
Referenced in my comment on pull request #43 in `code/api/src/modules/user/model.js`